vtm-android-example/src/org/oscim/android/test/MBTilesBitmapMapActivity.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2013 Hannes Janetzek
+ * Copyright 2016-2018 devemux86
+ * Copyright 2019 Andrea Antonello
+ *
+ * This file is part of the OpenScienceMap project (http://www.opensciencemap.org).
+ *
+ * This program is free software: you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ * PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along with
+ * this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.oscim.android.test;
+
+import android.app.AlertDialog;
+import android.os.Bundle;
+import android.os.Environment;
+import android.view.Menu;
+import android.view.MenuItem;
+
+import org.oscim.android.cache.TileCache;
+import org.oscim.android.tiling.MBTilesBitmapTileSource;
+import org.oscim.core.MapPosition;
+import org.oscim.layers.tile.bitmap.BitmapTileLayer;
+import org.oscim.tiling.source.UrlTileSource;
+import org.oscim.tiling.source.bitmap.DefaultSources;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+
+/**
+ * An example activity making use of mbtiles.
+ */
+public class MBTilesBitmapMapActivity extends MapActivity {
```

Comment:
Can you name that `MBTilesBitmapTileActivity`, to be similar to `BitmapTileActivity`?

Reply:
Done